docs/docs/build/bob-sdk/gateway.md

```diff
@@ -2,164 +2,135 @@
+This locks in the quote, placing a hold on the LP's funds. Pass the same `quoteParams` as before and the `quote` returned from the previous step.
 
-```ts title="/src/utils/gateway.ts"
-const orderId = await gatewayClient.createOrder(
-  onrampAddress,
-  evmAddress,
-  amount
-);
+Returns a `uuid` for the order and `psbtBase64`, a partially-signed Bitcoin transaction (PSBT) the user must sign.
+
+```ts
+const { uuid, psbtBase64 } = await gatewaySDK.startOrder(quote, quoteParams);
 ```
 
-### Send BTC
+### Sign the Bitcoin Transaction
 
-Create a Bitcoin transaction that sends the quoted `amount` of BTC from the user's wallet (`fromAddress`) to the LP's `bitcoinAddress`. This also publishes the user's `evmAddress` in the `OP_RETURN` of the transaction so the Gateway knows where to send wrapped BTC.
+Create a Bitcoin transaction that sends the quoted `amount` of BTC to the LP's `bitcoinAddress`. This also publishes a hash of the order's parameters in the `OP_RETURN` of the transaction so the Gateway can trustlessly verify the order on BOB.
 
 :::tip Connecting to Bitcoin wallets
-We recommend using our [sats-wagmi](./sats-wagmi.md) package to query your user's Bitcoin wallet address.
+We recommend using our [sats-wagmi](./sats-wagmi.md) package to interact with your user's Bitcoin wallet.
 :::
 
-```ts title="/src/utils/gateway.ts"
-import { createTransfer } from "@gobob/bob-sdk";
-import { AddressType, getAddressInfo } from "bitcoin-address-validation";
-import { Transaction } from '@scure/btc-signer';
-
-const tx = await createTxWithOpReturn(
-  fromAddress,
-  bitcoinAddress,
-  amount,
-  evmAddress
-);
-
-async function createTxWithOpReturn(
-  fromAddress: string,
-  toAddress: string,
-  amount: number,
-  opReturn: string,
-  fromPubKey?: string
-): Promise<Buffer> {
-  const addressType = getAddressInfo(fromAddress).type;
-
-  // Ensure this is not the P2TR address for ordinals (we don't want to spend from it)
-  if (addressType === AddressType.p2tr) {
-    throw new Error(
-      "Cannot transfer using Taproot (P2TR) address. Please use another address type."
-    );
-  }
-
-  // We need the public key to generate the redeem and witness script to spend the scripts
-  if (addressType === (AddressType.p2sh || AddressType.p2wsh)) {
-    if (!fromPubKey) {
-      throw new Error(
-        "Public key is required to spend from the selected address type"
-      );
-    }
-  }
-
-  const unsignedTx = await createTransfer(
-    "mainnet",
-    addressType,
-    fromAddress,
-    toAddress,
-    amount,
-    fromPubKey,
-    opReturn
-  );
-  const psbt = unsignedTx.toPSBT(0);
-
-  const signedTx = Transaction.fromPSBT(psbt);
-
-  return Buffer.from(signedTx.extract())
-}
+```ts
+import { base64 } from "@scure/base";
+import { Transaction } from "@scure/btc-signer";
+
+// NOTE: It is up to your implementation to sign the PSBT here!
```

Comment:
I know it's not ready yet but we should really detail how to also integrate sats-wagmi as a default example to do this

Reply:
Agreed. Let's add the sats-wagmi example asap in another PR.